espnet/nets/pytorch_backend/e2e_asr_mulenc.py

```diff
@@ -0,0 +1,612 @@
+#!/usr/bin/env python3
+
+"""Define e2e module for multi-encoder network."""
```

Comment:
Put your paper URL in this module docstring and E2E docstring.

Reply:
Thanks for the suggestion! Will do